### PR TITLE
updated with my name

### PR DESCRIPTION
**Pull request by anirranjan**

### CODE REVIEW

**Review by harshasrikara — CHANGES_REQUESTED** [review]
Code looks good! Can you change `8123` back to `8079` in the `index.ts` file?

File: functions/src/index.ts [hunk 1]
```diff
@@ -17,15 +17,19 @@ app.get("/hello", (request: Request, response: Response) => {
...
+app.get("/nirranjan-akilan", nirranjan_akilan);
 /* Change things above this line */
 
 //the line of code below will deploy to firebase cloud functions instead of running locally
 //leave this line commented
 //export const api = functions.https.onRequest(app);
 
 //the line of code below will run this locally
-app.listen(8079);
+app.listen(8123);
```
Comment (ez314) [inline functions/src/index.ts]:
```suggestion
app.listen(8079);
```

**Review by ez314 — CHANGES_REQUESTED** [review]
Update on PR

**Comment by anirranjan** [comment]
I have changed the port from 8123 to 8079.

**Review by ez314 — APPROVED** [review]

**Review by harshasrikara — APPROVED** [review]

**Pull request merged**